src/org/parosproxy/paros/CommandLine.java

```diff
@@ -328,6 +332,28 @@ private boolean parseKeywords(String[] args, int i) throws Exception {
                 this.configs.put(pair.substring(0, eqIndex), pair.substring(eqIndex + 1));
                 result = true;
             }
+        } else if (checkPair(args, CONFIG_FILE, i)) {
+            String conf = keywords.get(CONFIG_FILE);
+            File confFile = new File(conf);
+            if (! confFile.exists()) {

```

Comment:
(Not a Java developer ;)) Would `!confFile.isFile()` be a better check here (since `exists` and `canRead` would return true for a readable directory)?